File: warehouse-system/src/main/java/com/who/warehousesystem/dto/WbDetailsDto.java
package com.who.warehousesystem.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class WbDetailsDto {

    @JsonProperty(value = "wbNo")
    private Integer formNo;
    @JsonProperty(value = "wbDate")
    private LocalDate exportDate;
    @JsonProperty(value = "boxes")
    private Integer totalBoxes;
    @JsonProperty(value = "pallets")
    private Integer totalPallets;
    @JsonProperty(value = "city")
    private String healthCenterDistrictCityArName;
    @JsonProperty(value = "district")
    private String healthCenterDistrictArName;
    @JsonProperty(value = "center")
    private String healthCenterArName;
}

File: WarehouseSystem/WarehouseSystem/Dto/WbDetailsDto.cs
using Newtonsoft.Json;

namespace WarehouseSystem
{
    class WbDetailsDto
    {
        public string wbNo { get; set; }
        public string wbDate { get; set; }
        public string boxes { get; set; }
        public string pallets { get; set; }
        public string city { get; set; }
        public string district { get; set; }
        public string center { get; set; }

        public WbDetailsDto (string wbNo,string wbDate,string boxes,string pallets,string city,string district,
            string center)
        {
            this.wbNo = wbNo;
            this.wbDate = wbDate;
            this.boxes = boxes;
            this.pallets = pallets;
            this.city = city;
            this.district = district;
            this.center = center;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
